service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2864,10 +2864,14 @@ components:
           description: |
             SQL string against the source tables.
             This will be populated if source queries are configured for this entity.
+        entityIdSql:
```

Comment:
Should this go in `EntityOutputPreviewList` since it's not unique per entity?

Reply:
Good point. Yes, it isn't unique so makes sense to move it to EntityOutputPreviewList

Reply:
moved